lte/gateway/c/oai/lib/s6a_proxy/S6aClient.cpp

```diff
@@ -88,17 +90,63 @@ static bool read_mme_cloud_subscriberdb_enabled(void) {
   return mconfig.cloud_subscriberdb_enabled();
 }
 
-S6aClient& S6aClient::get_instance() {
-  static S6aClient client_instance;
-  return client_instance;
+S6aClient& S6aClient::get_s6a_proxy_instance(bool enable_s6a_proxy_channel) {
+  static S6aClient s6a_proxy_instance(enable_s6a_proxy_channel);
+  return s6a_proxy_instance;
```

Comment:
Why do we need `enable_s6a_proxy_channel` as a boolean argument here. Isn't it always true for s6a_proxy_instance?
```suggestion
S6aClient& S6aClient::get_s6a_proxy_instance() {
  static S6aClient s6a_proxy_instance(true);
  return s6a_proxy_instance;
```